src/main/java/it/gov/pagopa/payhub/ionotification/service/ioservice/IOServiceSearchServiceImpl.java

```diff
@@ -20,10 +23,32 @@ public IOServiceSearchServiceImpl(IORestConnector connector) {
         this.connector = connector;
     }
 
+
+    private ServicesListDTO getAllServicesWithPagination() {
```

Comment:
why "withPagination"?
who is calling the method is not giving the page settings... so who will call it will getAll

let's also return just the inner list
List<ServicePaginatedResponseDTO>